bigbluebutton-html5/imports/ui/components/poll/live-result/styles.scss

```diff
@@ -0,0 +1,110 @@
+@import "/imports/ui/stylesheets/variables/_all";
+
+:root {
+  --stats-border-color: #d4d9df;
```

Comment:
I would suggest you to prefix those variables with `poll-` since they are global.